cmd/tailscaled/tailscaled.go

```diff
@@ -166,6 +166,10 @@ func main() {
 func run() error {
 	var err error
 
+	if runtime.GOOS == "windows" {
+		os.Setenv("TS_LOG_TARGET", getRegString("LogTarget", ""))
```

Comment:
using environment variables as an internal mechanism for passing optional arguments is a bit gross.

It's effectively a global variable, so let's just be honest and use a global variable in the `logpolicy` package. You'll then have to think about the precedence there between the global variable and the environment variable.

But why not just have the logpolicy package do the registry lookup instead of all clients of logpolicy like tailscaled?

Reply:
Good idea, I'll do that.